include/battle.h

```diff
@@ -344,6 +344,15 @@ struct BattleTvMovePoints
     s16 points[2][PARTY_SIZE * 4];
 };
 
+struct MultiPartnerEnigmaBerry
+{
+    u8 versionSignatureLo;
+    u8 versionSignatureHi;
+    u8 vsScreenHealthFlagsLo;
+    u8 vsScreenHealthFlagsHi;
+    struct BattleEnigmaBerry battleEnigmaBerry;
+};
```

Comment:
Why is this struct called `MultiPartnerEnigmaBerry` if only one of its fields has anything to do with the Enigma Berry?